src/pyOpenMS/tests/unittests/test000.py

```
@@ -3539,6 +3539,48 @@ def testMSExperiment():
+    assert ms2.getMaxRT() == 200.0
+    assert ms2.getMinMZ() == 50.0
+    assert ms2.getMaxMZ() == 800.0
+
+    levels = sr.getMSLevels()
+    assert 1 in levels
+    assert 2 in levels
+
+    threw = False
+    try:
+        sr.byMSLevel(99)
+    except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)